cmd/ipfs/daemon.go

```diff
@@ -832,6 +832,12 @@ func serveHTTPGateway(req *cmds.Request, cctx *oldcmds.Context) (<-chan error, e
 		fmt.Printf("Gateway server listening on %s\n", listener.Multiaddr())
 	}
 
+	if cfg.Gateway.ExposeRoutingAPI.WithDefault(config.DefaultExposeRoutingAPI) {
+		for _, listener := range listeners {
+			fmt.Printf("Routing V1 API exposed at http://%s/routing/v1\n", listener.Addr())
+		}
+	}
```

Comment:
:information_source:  we do this for WebUI URL, would not hurt here – makes it easy for user to eyeball if it is enabled, and copy URL for testing.